src/AppBundle/GraphQL/Types/CharacterType.php

```diff
@@ -50,4 +61,101 @@ public function getDisplayName(): string
     {
         return $this->characterEntity->getDisplayName();
     }
+
+    /**
+     * Get character level
+     * @return int
+     */
+    public function getLevel(): int
+    {
+        return $this->characterEntity->getLevel();
+    }
+
+    /**
+     * Get character attack
+     * @return int
+     */
+    public function getAttack(): int
+    {
+        return $this->characterEntity->getAttack();
+    }
+
+    /**
+     * Get character defense
+     * @return int
+     */
+    public function getDefense(): int
+    {
+        return $this->characterEntity->getDefense();
+    }
+
+    /**
+     * Gets current health
+     * @return int
+     */
+    public function getHealth(): int
+    {
+        return $this->characterEntity->getHealth();
+    }
+
+    /**
+     * Gets maximum health
+     * @return int
+     */
+    public function getMaxHealth(): int
+    {
+        return $this->characterEntity->getMaxHealth();
+    }
+
+
+    /**
+     * Returns base stats
+     * @return array
+     */
+    private function returnBaseStats(): array
+    {
+        $stats = [
+        ];
+
+        $eventData = $this->getGameObject()->getEventManager()->publish(
+            "h/lotgd/crate-graphql/characterStats/public",
+            EventContextData::create(["character" => $this->characterEntity, "value" => $stats])
+        );
+
+        $stats = $eventData->get("value");
+
+        return $stats;
+    }
+
+    /**
+     * Returns public stats
+     * @return array
+     */
+    public function getPublicStats(): array
+    {
+        $stats = $this->returnBaseStats();
+
+        return $stats;
+    }
+
+    /**
+     * Returns public and private stats.
+     * @return array
+     */
+    public function getPrivateStats(): array

```

Comment:
I dont like that getPrivateStats() returns both public and private. Either change it to getPublicAndPrivateStats or have it return only the private stats.